### ISSUE TITLE
Inline aim is not working in paperspace notebooks or StudioLab Sagemeaker AWS notebooks

### ISSUE THREAD

**Issue by ummadiviany** [aimhubio/aim#1709]
## 🐛 Bug

I am trying to use aim inline UI in AWS Sagemaker Notebooks. But I am not able to see the UI. 

### To reproduce

Platform : StudioLab Sagemaker AWS Notebook

### Expected behavior
The aim UI has to load, instead refused to connect is showing.

### Environment

- Aim Version : 3.8.1
- Python version : 3.9.10
- pip version :   21.2.4
- OS (e.g., Linux) : Ubuntu 21.04
- Any other relevant information

### Additional
The used test code and resulting output is shown in the below image.

![Screenshot 2022-04-28 163823_AIM_ISSUE](https://user-images.githubusercontent.com/29988096/165740895-69888fdb-e4c3-431b-a12a-e66b99009241.png)

**Comment by rubenaprikyan** [comment]
Hi @ummadiviany, thanks fo the question.
I need to have a little exploration for the StudioLab, will get back here soon. Till I will be back, please go over this [guide](https://aimstack.readthedocs.io/en/latest/using/sagemaker_notebook_ui.html), may be helpful here.

**Comment by rubenaprikyan** [comment]
@ummadiviany  And to run aim ui on notebook, you need to pass your notebook' proxy url as `--proxy-url`  argument of `%aim up` command.

**Comment by ummadiviany** [comment]
Thanks @rubenaprikyan for pointing out. But its still not working out for me. Attaching a screenshot here.
![image](https://user-images.githubusercontent.com/29988096/165744593-b48b98f3-820b-4fc0-992b-e9e01f59df12.png)

**Comment by rubenaprikyan** [comment]
@ummadiviany have you installed `jupyter-server-proxy` package on your notebook instance from the source(aim forked repository as mentioned in the guide) ?

**Comment by rubenaprikyan** [comment]
@ummadiviany it need some time to approve the new account on SageMaker StudioLab, I can do some tests once they will approve the account. Until it please make sure you did everything right for the proxy server.

**Comment by ummadiviany** [comment]
> @ummadiviany have you installed `jupyter-server-proxy` package on your notebook instance from the source(aim forked repository as mentioned in the guide) ?

@rubenaprikyan Yes I have installed `jupyter-server-proxy` but still facing the same error.

**Comment by ummadiviany** [comment]
> @ummadiviany it need some time to approve the new account on SageMaker StudioLab, I can do some tests once they will approve the account. Until it please make sure you did everything right for the proxy server.

It will take a ~week to accept and verify your account.

**Comment by rubenaprikyan** [comment]
Hi @ummadiviany, sorry for the late reply.
There is an issue while using notebook extension in StudioLab.

Here are steps to run aim  ui on StudioLab using terminal, try to run with terminal until we will fix extension' issue on StudioLab.
- set `__AIM_PROXY_URL__` env variable to `https://<instance>.studio.<region>.sagemaker.aws`
- $aim up --base-path /studiolab/default/jupyter/proxy/absolute/43800/aim-sage 
- open printed Proxy url in a new browser tab

**Comment by ummadiviany** [comment]
@rubenaprikyan Yes it finally working through terminal. But a slight change in the printed URL is required.
Here are the final steps.
In terminal
1. $ set __AIM_PROXY_URL__ = "https://<instance>.studio.<region>.sagemaker.aws"
2. $ aim up --base-path /studiolab/default/jupyter/proxy/absolute/43800/aim-sage
3. It will print an url like this  **http://127.0.0.1:43800/studiolab/default/jupyter/proxy/absolute/43800/aim-sage**
4. Change that url to  **https://<instance>.studio.<region>.sagemaker.aws/studiolab/default/jupyter/proxy/absolute/43800/aim-sage**
5. Open this modified url in new tab, it will work.

Shall we close this now??
